package.json

```diff
@@ -219,7 +219,7 @@
     "@fortawesome/pro-regular-svg-icons": "^6.1.0",
     "@fortawesome/pro-solid-svg-icons": "^6.1.0",
     "@fortawesome/pro-thin-svg-icons": "^6.4.0",
-    "@kinde-oss/kinde-auth-pkce-js": "^3.0.27",
+    "@kinde-oss/kinde-auth-pkce-js": "^4.2.0",
```

Comment:
I couldn't see breaking changes.. I didn't actually test that this works because I don't think there's an available route to test it currently